action/action-dsl/src/main/java/com/powsybl/action/dsl/afs/ActionScript.java

```diff
@@ -55,4 +59,19 @@ public ActionDb load(Network network) {
     public List<Contingency> getContingencies(Network network) {
         return new ArrayList<>(load(network).getContingencies());
     }
+
+    @Override
+    public void writeScript(String content) {
```

Comment:
We can probably add a protected implementation of `writeScript` in `AcstractScript` to share code between `ActionScript` and `ModificationScript`.
Probably `AbstractScript` class is not well named: we do not understand that it's a read only script without the class hierarchy.